doc/charm++/manual.rst

```diff
@@ -8104,7 +8104,10 @@ call to initiate the checkpoint.
 
 After ``CkStartCheckpoint`` is executed, a directory of the designated
 name is created and a collection of checkpoint files are written into
-it.
+it. Note that checkpoints are written to and read from several
```

Comment:
Could maybe change this to a `.. note::`.

Reply:
Added.